ledger/src/blockstore_db.rs

```diff
@@ -1468,6 +1468,31 @@ pub struct WriteBatch {
     write_batch: RWriteBatch,
 }
 
+pub struct BlockstoreByteReference<'a> {
+    slice: DBPinnableSlice<'a>,
+}
```

Comment:
why do we need this wrapper type?
can't we just return `DBPinnableSlice`?
or just define a type synonym 
```rust
type BlockstoreByteReference = DBPinnableSlice;
```
?